AEPMessaging/Sources/Inbound.swift

```diff
@@ -107,14 +99,25 @@ public class Inbound: NSObject, Codable {
 }
 
 public extension Inbound {
+    static func from(consequenceDetail: [String: Any]?, id: String) -> Inbound? {
+        guard let consequenceDetail = consequenceDetail else {
+            return nil
+        }
+
+        var consequenceDetailData = consequenceDetail
```

Comment:
nit: don't need `consequenceDetailData` if we do `guard var consequenceDetail...` on line 103